.travis.yml

```diff
@@ -27,8 +27,8 @@ script:
 - if [ "$TRAVIS_RUST_VERSION" == "nightly" ]; then cargo check --benches --manifest-path=benches/Cargo.toml; fi
 # Make sure `no_std` version checks.
 - if [ "$TRAVIS_RUST_VERSION" == "nightly" ]; then cargo no-std-check --no-default-features --features core; fi
-# Check that `vec_memory` feature works.
-- cargo check --features vec_memory
+# Check that `virtual_memory` feature works.
+- cargo check --features virtual_memory
```

Comment:
I think we can just delete this whole file (in another PR). We only use github actions. So no changes needed here.

Reply:
will do in a micro follow-up PR

Reply:
Resolved in https://github.com/paritytech/wasmi/pull/273